.github/workflows/smoke-test.yml

```diff
@@ -0,0 +1,40 @@
+name: smoke test
+
+on:
+  pull_request: ~
+  workflow_dispatch:
+  schedule:
+    - cron: 0 22 * * *
+
+jobs:
+  smoke-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: pnpm/action-setup@v3
+        with:
+          version: ${{ vars.PNPM_VERSION }}
+      - uses: actions/setup-node@v4
+        with:
+          node-version: ${{ vars.DEFAULT_NODE_VERSION }}
+          cache: 'pnpm'
+      - run: pnpm i --frozen-lockfile
+      # - name: Create .env file
```

Comment:
I suppose this can be removed, since the binding will always be in the VCAP variables?